x/distribution/spec/03_begin_block.md

```diff
@@ -0,0 +1,80 @@
+<!--
+order: 3
+-->
+
+# Begin Block
+
+At each `BeginBlock`, the fees received in the previous block are transferred to the distribution `ModuleAccount`, as it's the account the one who keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve community tax is then charged, then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Note the social distribution is applied to proposer validator in addition to the proposer reward.
+
+The amount of proposer reward is calculated from pre-commits Tendermint messages in order to incentivize validators to wait and include additional pre-commits in the block. All provision rewards are added to a provision reward pool which validator holds individually (`ValidatorDistribution.ProvisionsRewardPool`).
```

Comment:
```suggestion
The block proposer reward is calculated from Tendermint pre-commit messages in order to incentivize validators to wait and include additional pre-commits in the block. All provision rewards are added to a provision reward pool which validator holds individually (`ValidatorDistribution.ProvisionsRewardPool`).
```

Reply:
I'm not sure what is meant by 'provision rewards'. I didn't write these paragraphs, they only appear as new text because the file changed name from `03_end_block.md` to `03_begin_block.md`. I'm hesitant to remove the existing text since it may use lingo that is in popular use but that I'm unfamiliar with. I'm happy to remove the paragraphs, or just shorten them down (as suggested below) if they only seem to cause confusion.

Reply:
Validator takes a provision (commission), eg 5%, for his work. So he is not distributing all rewards - he will firstly take the provision, and then distribute the remaining (eg 95%) to delegators.